### ISSUE TITLE
[EN][Dun Morogh] List of Coldridge Valley bugs

### ISSUE THREAD

**Issue by Petrauskas** [Atlantiss/BugTracker#1385]
1. Quest [Make Hay While the Sun Shines](http://www.wowhead.com/quest=24486) should only be obtainable after turning in quest [Dwarven Artifacts](http://www.wowhead.com/quest=24477). 

2. Quest [All the Other Stuff](http://www.wowhead.com/quest=24475) should only be obtainable after turning in quest [First Things First: We're Gonna Need Some Ale](http://www.wowhead.com/quest=24474).

3. There are too few mobs called [Frostmane Blade](http://www.wowhead.com/npc=37507) than there should be spawned inside the cave in Coldridge Valley which are required for quest [Ice and Fire](http://www.wowhead.com/quest=218), thus making it a pain to complete if somebody else has recently done it as their respawn times are quite long. 

4. In Coldridge Valley (also it seems that in entirety of Dun Morogh as well) mobs have a very high chance of dropping a 6 slot bag. Maybe intended for newcomers, although it wasn't like this in human starting area.

5. [Frostmane Troll Whelps](http://www.wowhead.com/npc=706) should be neutral attackable, not hostile attackable.

6. [Ragged Young Wolves](http://www.wowhead.com/npc=705) are not marked as quest NPCs for [All the Other Stuff](http://www.wowhead.com/quest=24475) nor have information in their tooltips that they drop [Ragged Wolf Hide](http://www.wowhead.com/item=49748) required for the above mentioned quest. They do drop the items, just that the information isn't there. 

7. NPC [Milo Geartwinge](http://www.wowhead.com/npc=37113) and his gyrocopter despawn after offering the quest [Pack Your Bags](http://www.wowhead.com/quest=24492). Apart from being a visual bug, it also means that If the player declines it for whatever reason he won't be able to pick up further quests in the chain as you get just one chance to accept that quest. He does reapper if you accept and complete the objective for the above mentioned quest.

8. [Coldridge Citizen's](http://www.wowhead.com/npc=37218) gossip text is missing.

**Comment by Navedonar** [comment]
Frostmane Troll Whelps should be neutral attackable, not hostile attackable.
fixed

**Comment by CrazyFredy** [comment]
Yeah, now they are neutral talkable (though they can still be killed).
